babel/src/board.js

```diff
@@ -37,6 +37,44 @@ export function hasMatchInPairOfRows (pairOfRows) {
 };
 
 /**
+ * @private
+ * @description Used to provide more granularity to functions that need to call iterchunks
+ * on a non-transposed set of orbs, as well as a transposed set of orbs.
+ * @see findMatches
+ */
+let _iterchunks = (orbs, chunkLimitRange, includePositionInformation, isTransposed) => {
+    let chunks = [];
+    let [width, height] = chunkLimitRange;
+    let [finalPositionWidth, finalPositionHeight] = [orbs[0].length - width, orbs.length - height];
+    _.each(_.range(0, finalPositionHeight + 1), heightIndex => {
+        _.each(_.range(0, finalPositionWidth + 1), widthIndex => {
+            let chunkData = orbs.slice(heightIndex, heightIndex + height).map(row => {
+                return row.slice(widthIndex, widthIndex + width);
+            });
+
+            if (includePositionInformation) {
+                let startingCoordinates = [heightIndex, widthIndex];
+                let endingCoordinates = [heightIndex + height - 1, widthIndex + width - 1];
+                if (isTransposed) {
+                    startingCoordinates = startingCoordinates.reverse();
+                    endingCoordinates = endingCoordinates.reverse();
+                }
+
+                chunkData.push({
+                    position: {
+                        first: startingCoordinates,
+                        last: endingCoordinates
+                    }
+                });
+            }
+
+            chunks.push(chunkData);
+        });
+    });
+    return chunks;
+};
```

Comment:
I looked into this. It looks like you changed the names from `heightcoordinates, widthcordinates` to what is here.